docs/src/components/component/button/button.config.yml

```diff
@@ -0,0 +1,125 @@
+title: button
+collated: true
+variants:
+  - name: default
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: primary
+      buttonText: default
+  - name: base
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: base
+      buttonText: base
+  - name: warning
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: warning
+      buttonText: warning
+  - name: active
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: primary
+      buttonText: active
+  - name: disabled
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: cautious
+      buttonText: disabled
+  - name: finish
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: finish
+      buttonText: finish-outline
```

Comment:
fwiw, I like what you do with the swatches, there's the component itself, and then there's some helper text so you can see the name, as well as a realistic example.